etc/install_package_dependencies_for_virtualenv.sh

```diff
@@ -0,0 +1,38 @@
+# This script can be used to install all the required package dependencies on a clean system
+# Run this file as . install_package_dependencies_for_virtualenv.sh to retain the virtual environment after the execution of this file
+
+PROJECT_PATH=$(pwd)
+
+# Install Python-PIP
+sudo pip3 install --upgrade pip
+
+# Install virtualenv, create a virtual environment and activate it
+sudo pip3 install virtualenv
+virtualenv env
+source env/bin/activate
```

Comment:
@tejalwakchoure Ideally, the installation file would have commands upto this point. After this we should be able to just run 
```
sudo -H pip3 install -r requirements.txt
```
to install all the required dependencies.

With virtualenv option, I see a need to remove the above command from [install_package_dependencies.sh](https://github.com/DeveloperCAP/MLCAT/blob/master/etc/install_package_dependencies.sh). 

One way to think about installation is:
```
install_os_packages.sh     #performs deb / direct package installations
install_virtualenv.sh           #runs required commands for setting up virtualenv
install.sh                             #performs direct installation of pip packages to base OS
```

@tejalwakchoure you can take the help of @AakankshaSanctis in order to put the necessary scripts in place.

Reply:
I think separating all the installation packages won't be necessary, as virtualenv is preferred over base OS installation, better to change the installations to virtualenv and have a simpler installation process